docs/design/services/dynamodb/high-level-library/prototype/option-1/sync/Prototype.java

```diff
@@ -0,0 +1,1026 @@
+         *     <li>{@link File} -> {@link ItemAttributeValueType#BYTES}</li>
+         *     <li>{@link Boolean} -> {@link ItemAttributeValueType#BOOLEAN}</li>
+         *     <li>{@link Collection} -> {@link ItemAttributeValueType#LIST_OF_*}</li>
+         *     <li>{@link Stream} -> {@link ItemAttributeValueType#LIST_OF_*}</li>
+         *     <li>{@link Iterable} -> {@link ItemAttributeValueType#LIST_OF_*}</li>
+         *     <li>{@link Iterator} -> {@link ItemAttributeValueType#LIST_OF_*}</li>
+         *     <li>{@link Enumeration} -> {@link ItemAttributeValueType#LIST_OF_*}</li>
+         *     <li>{@link Optional} -> {@link ItemAttributeValue#*}</li>
+         *     <li>{@link Map} -> {@link ItemAttributeValueType#ITEM}</li>
+         *     <li>{@link Object} -> {@link ItemAttributeValueType#ITEM}</li>
+         *     <li>{@link null} -> {@link ItemAttributeValueType#NULL}</li>
+         * </ul>
+         *
+         * Usage Example:
+         * <code>
+         *     DocumentClientConfiguration clientConfiguration =
+         *             DocumentClientConfiguration.builder()
+         *                                        .addConverter(InstantsAsStringsConverter.create())
+         *                                        .build();
+         *
+         *     try (DynamoDbDocumentClient client = DynamoDb.documentClientBuilder()
+         *                                                  .documentClientConfiguration(clientConfiguration)
+         *                                                  .build()) {
+         *
+         *         Table table = client.getTable("my-table");
+         *         UUID id = UUID.randomUUID();
+         *         table.putItem(Item.builder()
+         *                           .putAttribute("partition-key", id)
+         *                           .putAttribute("creation-time", Instant.now())
+         *                           .build());
+         *
+         *         Item item = table.getItem(Item.builder()
+         *                                       .putAttribute("partition-key", id)
+         *                                       .build());
+         *
+         *         // Items are usually stored as a number, but it was stored as an ISO-8601 string now because of the
+         *         // InstantsAsStringsConverter.
+         *         assert item.attribute("creation-time").isString();
+         *         assert item.attribute("creation-time").as(Instant.class).isBetween(Instant.now().minus(1, MINUTE),
+         *                                                                            Instant.now());
+         *     }
+         * </code>
+         */
+        DocumentClientConfiguration.Builder converters(List<ItemAttributeValueConverter<?>> converters);
+        DocumentClientConfiguration.Builder addConverter(ItemAttributeValueConverter<?> converter);
+        DocumentClientConfiguration.Builder clearConverters();
```

Comment:
What is the difference between clearConverters and passing null/empty map to converters(..) method?

Reply:
No difference. This is just matching the standard we have for other setters in the SDK.